scrambler-interface/WebContent/scrambler-interface/js/ui.js

```diff
@@ -11,6 +11,8 @@ var mark2 = {};
 (function() {
     "use strict";
 
+    var THREE_MULTI_MAX_PUZZLES_PER_ATTEMPT = 28;
```

Comment:
nit: this sounds like 28 is the maximum number of scrambles we can generate per attempt, I'd probably change "MAX" to "DEFAULT" or something like that.

Reply:
Ah, very good point! I've changed it to `THREE_MULTI_DEFAULT_PUZZLES_PER_ATTEMPT`.